lib/utils/webpack.config.js

```diff
@@ -117,6 +117,7 @@ module.exports = (program, directory, stage, webpackPort = 1500, routes = []) =>
             },
             __PREFIX_LINKS__: program.prefixLinks,
           }),
+          new ExtractTextPlugin('styles.css'),
```

Comment:
Without this line the following snippet:

`import styles from './style.css'`

is producing empty object. I don't fully understand the reason of it. 

Maybe there is a better alternative for this?